docs/source/NeuralNetworkTheory.rst

```diff
@@ -107,10 +107,10 @@ Python File: https://colab.research.google.com/drive/1hafIuEqEhiioZpQajJWUt6aSsP
 In simple terms, a neural network is a function with the particular ability to learn to predict complex patterns using
 data.
 
-.. image:: ~Image/NeuralNetworkbyAndrewNg.png
+.. image:: Image/NeuralNetworkbyAndrewNg.png
```

Comment:
Thank you! Sorry, but it would be much better with name `figs`, `img`, or `images`, not `image`. Thank you!